src/bootstrap/config.toml.example

```diff
@@ -152,6 +152,9 @@
 # known-good version of OpenSSL, compile it, and link it to Cargo.
 #openssl-static = false
 
+# Run the build with low priority
+#low_priority = false
```

Comment:
In TOML configuration we tend to prefer dashes, could this perhaps be `low-priority`?